appeals/web/src/server/appeals/appeal-details/representations/representations.service.js

```diff
@@ -15,3 +17,19 @@ export const getRepresentationCounts = (apiClient, appealId, statusFilter = 'all
 
 	return apiClient.get(url).json();
 };
+
+/**
+ * Fetch paginated appeal comments based on appeal ID and status.
+ *
+ * @param {import('got').Got} apiClient
+ * @param {string} appealId
+ * @param {string} type
+ * @returns {Promise<Representation>}
+ */
+export const getSingularRepresentationByType = async (apiClient, appealId, type) => {
+	let url = `appeals/${appealId}/reps?type=${type}`;
```

Comment:
very nitpicky: this might as well be `const`

Reply:
Oh yeah definitely, it was just a let in the place I copied it from. Surprised we don't have a lint rule that catches that actually